### PR TITLE
Add dark mode option to info modal. 

### PR DESCRIPTION
**Pull request by iandoesallthethings**
This is just a quick and dirty darkmode using the built-in tailwind class method. I added a couple more classes to tailwind.config.js. The color can be tweaked there as well as in the customStyles object. 

It's toggled with a simple checkbox in the info modal and the preference is saved to local storage. 

Hope this helps! :D

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -69,6 +69,9 @@ function App() {
     setInfoModalIsOpen(false)
   }
 
+  const [darkMode, setDarkMode] = useLocalStorage('dark-mode', false)
+  const toggleDarkMode = () => setDarkMode(!darkMode)
```
Comment (octokatherine) [inline src/App.js:73]:
```suggestion
  const toggleDarkMode = () => setDarkMode(prev => !prev)
```

I _think_ this is the safer way to update state when relying on the previous value of it

File: src/index.css [hunk 2]
```diff
@@ -17,20 +17,13 @@ body {
...
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
-#root,
-body,
-html {
-  /* Prevent overscroll on chrome (not supported on safari) */
-  overscroll-behavior-y: none;
-}
-
```
Comment (octokatherine) [inline src/index.css:33]: I don't think we want to be deleting these lines

**Review by octokatherine — CHANGES_REQUESTED** [review]
Thanks for the contribution! I left a couple suggestions inline. Also, I don't think we want the `package-lock` changes.

**Review by iandoesallthethings — COMMENTED** [review]
Oops, didn't mean to commit the package-lock changes! Everything else looks good to me. :)

File: src/index.css (hunk 2 above)
Comment (iandoesallthethings) [reply src/index.css:33]: Wait, do I need to roll back this line manually? This is my first pull request, sorry. :p

File: src/index.css [hunk 3]
```diff
@@ -17,20 +17,13 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   box-sizing: border-box;
-  background-color: hsl(231, 16%, 92%);
+  /* background-color: hsl(231, 16%, 92%); */
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
```
Comment (octokatherine) [inline src/index.css:26]:
```suggestion
#root,
body,
html {
  /* Prevent overscroll on chrome (not supported on safari) */
  overscroll-behavior-y: none;
}
```

File: src/App.js [hunk 4]
```diff
@@ -320,99 +357,123 @@ function App() {
...
+            <button
+              className="absolute top-4 right-4 rounded-full nm-flat-background dark:nm-flat-background-dark text-primary dark:text-primary-dark p-1 w-6 h-6 sm:p-2 sm:h-8 sm:w-8"
+              onClick={handleInfoClose}
             >
-              <Github />
-            </a>
+              <Close />
+            </button>
+            <div className="h-full flex flex-col items-center justify-center max-w-[390px] mx-auto pt-9 text-primary dark:text-primary-dark">
+              <div className="flex-1 w-full sm:text-base text-sm">
```
Comment (octokatherine) [inline src/App.js:422]:
```suggestion
              <div className="flex-1 w-full border sm:text-base text-sm">
```

**Pull request merged**